collect_app/src/androidTest/java/org/odk/collect/android/regression/FillBlankFormTest.java

```diff
@@ -350,7 +351,7 @@ public void values_ShouldBeRandom() {
         for (int i = 1; i <= 3; i++) {
             FormEntryPage formEntryPage = new MainMenuPage().startBlankForm("random test");
             formEntryPage.inputText("3");
-            formEntryPage.swipeToNextQuestion();
+            formEntryPage.swipeToNextQuestion("random test");
```

Comment:
Here question text is `Your random number was`

Reply:
Same here, when I changed text, then I get `NoMatchingViewException`.

Reply:
Ok I've fixed this.